dashboard/test/models/lti_user_identity_test.rb

```diff
@@ -6,4 +6,8 @@ class LtiUserIdentityTest < ActiveSupport::TestCase
     refute build(:lti_user_identity, user: nil).valid? "user is required"
     refute build(:lti_user_identity, lti_integration: nil).valid? "lti_integration is required"
   end
+  test "should not allow duplicate subject/lti_integration_id" do
+    lti_user_identity = create(:lti_user_identity)
+    refute build(:lti_user_identity, lti_integration_id: lti_user_identity.lti_integration.id, subject: lti_user_identity.subject).valid?
```

Comment:
Please test that the record is invalid specifically because of the uniqueness check failure, and not due to any other validation error.